### ISSUE TITLE
Add edit_url to JSON

### ISSUE THREAD

**Issue by joshreisner** [code4recovery/12-step-meeting-list#438]
the edit_url parameter is part of the JSON spec now and adding that to WordPress JSON feeds will enable TSML UI (and perhaps Meeting Guide, someday) to send better feedback emails to admins. Example:

```
  "edit_url": "https://demo.code4recovery.org/wp-admin/post.php?post=5888&action=edit",
```

This is in service of [this request](https://github.com/code4recovery/tsml-ui/issues/127).

**Comment by district43aa** [comment]
Where do I find the URL with JSON in Wordpress to give for another site to display our meetings on theirs?

**Comment by tech2serve** [comment]
Please [see](https://github.com/code4recovery/12-step-meeting-list/issues/42#issuecomment-922336909)

**Comment by district43aa** [comment]
Oh I did find that and copied and sent the URL when I clicked on it. This is what came up:https://district43aa.org/2020/wp-admin/admin-ajax.php?action=meetingsDoes this look correct and would it work or should I be converting it somehow to a .json format as I was ask for by our Area website chair. This is what our area website chair asked for:For us to display the meetings from District 43 on our site we need the meeting guide cache URL from your site. It’s located on their website in:  *Wordpress-root-directory*/wp-admin/ And the file naming convention should be something like "tsml-cache-*unique-alpha-numeric-string*.json" So, for example, Victoria’s file is located at:Https://aavictoria.ca/wordpress/wp-admin/tsml-cache-be40030f75.json The BCYUKON file is located at:Https://bcyukonaa.org/wordpress/wp-admin/tsml-cache-8118ba7e10.json  Where do I get this format?
    	Get Outlook for iOS
    
  

 From: Tech2Serve ***@***.***>Sent: Saturday, September 18, 2021 9:37 AMTo: code4recovery/12-step-meeting-listCc: district43aa; CommentSubject: Re: [code4recovery/12-step-meeting-list] Add edit_url to JSON (#438) 
Please see

**Comment by tech2serve** [comment]
The feed you included is correct (it is JSON). You can test it by putting it in your browser and you should see a list of meetings formatted in JSON.

**Comment by joshreisner** [comment]
reopening

**Comment by joshreisner** [comment]
shipped in version `3.14.3`
